### DISCUSSION TITLE
Globbing possible in .ansible-lint-ignore?

### DISCUSSION THREAD

**Discussion by runofthemill** [ansible/ansible-lint#3165] [Q&A] [answered]
Reading through the `.ansible-lint` docs I saw this:

> We discourage the use of skip_list below because that will hide violations from the output. When putting ignores inside the ignore file, they are marked as ignored, but still visible, making it easier to address later.

That makes a lot of sense, and so I wanted to move this out of my `.ansible-lint` file, so the violation is ignored but still visible:

```yaml
skip_list:
  - name[casing]
```

I tried a few things but couldn't figure out a way to glob in `.ansible-lint-ignore` - is it possible, or would I have to list each violating file on it's own line?

Cheers!

**Comment by ssbarnea** [comment, marked as answer]
For the moment globbing is not possible, only explicit files are allowed. I am not sure if we will want to add support for globbing there. Keeping the file format extremely simple makes it easier to consume and even re-generate.

**Reply by runofthemill** [reply]
That makes sense! When I have a few moments I'll make a PR to clarify that in the docs. Cheers :)
